packages/inventory-compliance/src/Compliance.js

```diff
@@ -56,10 +57,10 @@ const SystemQuery = ({ data: { system }, loading, hidePassed }) => (
         <SystemRulesTable hidePassed={ hidePassed }
             system={ system }
             columns={ columns }
-            profileRules={ system?.profiles.map((profile) => ({
+            profileRules={ system?.profiles.map(({ id, refId, name, policy, rules }) => ({
```

Comment:
Writing it like this would avoid having to assign all properties:
```js
profileRules={ system?.profiles.map((profile) => ({
    system,
    profile,
    rules: profile.rules
})) } 
```

We don't really need to limit what passes through.

Reply:
The limitation was there before. I don't see a reason why not to include the whole profile object.

Reply:
Updated.
I guess the original idea was not passing the rules two different ways.